src/main/java/com/puppycrawl/tools/checkstyle/checks/design/DesignForExtensionCheck.java

```diff
@@ -175,45 +175,93 @@
  * <pre>
  * &lt;module name=&quot;DesignForExtension&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public abstract class Foo {
+ *   private int bar;
+ *
+ *   public int method1() {return 2;}  // Violation. No javadoc.
```

Comment:
Make all method name to be like m1, m2,...

We need code more horizontally compact to see violation comment on phone screen.

Reply:
Thank you! I will do that as soon as I get home!

Reply:
Hi @romani , I already resolve this one in commit [184f6da](https://github.com/zwwang98/checkstyle/commit/184f6da3f50b2d5e26cc0765fd165a23ae29f7e8).

Reply:
@zwwang98 allow reviewer to resolve all items as they come back to PR, simply reply "done" to each item as you complete it.

Reply:
> @zwwang98 allow reviewer to resolve all items as they come back to PR, simply reply "done" to each item as you complete it.

Thank you @nrmancuso , it is really helpful!

Reply:
done